sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/GalleryApplicationVersionPublishingProfile.cs

```diff
@@ -59,13 +59,17 @@ public GalleryApplicationVersionPublishingProfile()
         /// <param name="targetExtendedLocations">The target extended locations
         /// where the Image Version is going to be replicated to. This property
         /// is updatable.</param>
+        /// <param name="advancedSettings">Optional. Additional settings to
```

Comment:
You should add a description to this PR, explain what the feature is and what your tests do. Keep URLs out of the title, and put them in the description.

Reply:
Updated the PR title and description